### DISCUSSION TITLE
save settings option Yes or No - POLL

### DISCUSSION THREAD

**Discussion by Akossimon** [Bing-su/adetailer#684] [Polls]
if save settings option is something you like, please vote bellow, to show the devs and creators how many would love such a feature

**Comment by brknsoul** [comment]
Kind of already exists..
You can change the default on-startup settings of pretty much anything in the UI; Settings > Defaults

**Comment by Akossimon** [comment]
thats not correct...one setting is not a choice of anything worth saving. Goal is to save many “different” settings .

On 14 Aug 2024, at 23:56, brknsoul ***@***.***> wrote:


Kind of already exists..
You can change the default on-startup settings of pretty much anything in the UI; Settings > Defaults
